pipeline/beam_tables.py

```diff
@@ -485,8 +485,8 @@ def _add_metadata(
 
     # PCollection[Tuple[Tuple[date,ip],Dict[input_name_key,List[Row]]]]
     grouped_metadata_and_rows = (({
-        IP_METADATA_PCOLLECTION_NAME: ips_with_metadata,
-        ROWS_PCOLLECION_NAME: rows_keyed_by_ip_and_date
+        BEAM_COGROUP_A_SIDE: ips_with_metadata,
```

Comment:
Provide meaningful names instead of A and B.

Reply:
Done. I added a third global name

```
RECEIVED_IPS_PCOLLECTION_NAME = 'received_ips'
```

I like how this makes it clearer how the triple-cogroup is working, since it's clearer which if the three parts we're joining when.

The only issue is that we're currently using `merge_metadata_with_rows` for two if the cogroups, and to make that work with the different names I had to add a kvarg to that function. Not entirely sure I like it but let me know.